packages/client/src/modules/post/containers/Post.spec.js

```diff
@@ -327,31 +328,29 @@ describe('Posts and comments example UI works', () => {
     expect(content.find('.delete-comment')).has.lengthOf(6);
   });
 
-  step('Comment editing works', done => {
+  step('Comment editing works', async done => {
     mutations.editComment = (obj, { input }) => {
       expect(input.postId).to.equal(3);
       expect(input.content).to.equal('Edited comment 2');
       done();
       return input;
     };
-
     const editButtons = content.find('.edit-comment');
     expect(editButtons).has.lengthOf(6);
     editButtons.last().simulate('click');
-
+    editButtons.last().simulate('click');
     const commentForm = content.find('form[name="comment"]');
     expect(
       commentForm
         .find('[name="content"]')
         .last()
         .instance().value
-    ).to.equal('Post comment 2');
+    ).to.equal('Post comment 2'); //must be Post comment 2
```

Comment:
This comment is redundant. It is evident that it must equal `Post comment 2`, comment doesn't add new information, so it is redundant.